### PR TITLE
chore: Add viteconfig-timestamp to .gitignore

### PR DESCRIPTION
**Pull request by etnoy**
Ignore the spurious vite configs that sometimes pop up after testing
image

### CODE REVIEW

**Review by jrasm91 — APPROVED** [review]

File: .gitignore
Comment (danieldietzler) [inline .gitignore:1]: The files in your screenshot are all in `web/`, so you should also edit the gitignore file over there. In fact, I don't even think this works.

**Review by danieldietzler — COMMENTED** [review]
I have never run into those lol

File: .gitignore
Comment (jrasm91) [reply .gitignore:1]: Probably should just do **/...

Comment (danieldietzler) [reply .gitignore:1]: Or that, yea

Comment (etnoy) [reply .gitignore:1]: This works, all dangling config files were gone from the git list after this change. .gitignore does not need a leading **/

These files are sometimes left over after a failed e2e test. They're supposed to be cleaned up but sometimes this doesn't happen and I get annoyed. Hence this PR.

I still think this should be top-level

Comment (danieldietzler) [reply .gitignore:1]: > .gitignore does not need a leading **/

Oh ok, TIL!

**Review by danieldietzler — APPROVED** [review]

**Pull request merged**